.circleci/unittest/windows/scripts/set_cuda_envs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+if [ "${CU_VERSION:-}" != "cpu" ] ; then
```

Comment:
Nit: instead of having an if statement for the entire script, just exist early in case of CPU build
```suggestion
if [ "${CU_VERSION:-}" == "cpu" ] ; then
  exit 0
fi
```

Reply:
updated